.github/workflows/ci.yml

```diff
@@ -268,7 +268,14 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - id: "auth"
+      - name: Setup kubectl
+        id: install
+        uses: azure/setup-kubectl@v3
+        with:
+          version: "1.23.15"
```

Comment:
how it was installed before this change? as a dependency?

Reply:
The runner image `ubuntu-20.04` and `ubuntu-22.04` already contain kubectl in version `1.25.4` (see https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md and https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md).

This change should install 1.23.15 and use that version (we'll see in the logs).